packages/rum/test/benchmarks/analyzer.js

```diff
@@ -28,19 +28,48 @@ const stats = require('stats-lite')
 const { readFileSync } = require('fs')
 const path = require('path')
 const zlib = require('zlib')
+const webpack = require('webpack')
+const {
+  getWebpackConfig,
+  BUNDLE_TYPES
+} = require('../../../../dev-utils/build')
 const { runs, noOfImages } = require('./config')
 
 const dist = path.join(__dirname, '../../dist')
 
-function getMinifiedApmBundle() {
-  return readFileSync(
-    path.join(dist, 'bundles/elastic-apm-rum.umd.min.js'),
-    'utf-8'
-  )
+function customApmBuild(filename) {
+  /**
+   * Match it with the default webpack prod build of elasticApm
+   * expect the function names are not mangled
+   */
+  const config = {
+    entry: path.join(__dirname, '../../src/index.js'),
+    output: {
+      filename,
+      path: path.join(dist, 'bundles'),
+      library: '[name]',
+      libraryTarget: 'umd'
+    },
+    ...getWebpackConfig(BUNDLE_TYPES.BROWSER_ESM_PROD)
+  }
+
+  return new Promise((resolve, reject) => {
+    webpack(config, err => {
+      if (err) {
+        reject(err)
+      }
+      console.info('custom apm build - ', filename, 'generated')
+      resolve()
+    })
+  })
```

Comment:
Why this needs to be async?

Reply:
so we can await on the build part before starting the server.